### PR TITLE
mopidy-scrobbler: init at 2.0.1

### PR DESCRIPTION
**Pull request by jakeisnt**
###### Motivation for this change
[Mopidy-Scrobbler](https://github.com/mopidy/mopidy-scrobbler) is a Mopidy extensions maintained by the Mopidy organization. Many other Mopidy extensions maintained by them are already packaged.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by otavio — APPROVED** [review]

**Comment by lovesegfault** [comment]
Result of `nixpkgs-review pr 107993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mopidy-scrobbler</li>
  </ul>
</details>

File: pkgs/top-level/all-packages.nix [hunk 1]
```diff
@@ -22918,7 +22918,8 @@ in
     mopidy-spotify
     mopidy-spotify-tunigo
     mopidy-tunein
-    mopidy-youtube;
+    mopidy-youtube
+    mopidy-scrobbler;
```
Comment (lovesegfault) [inline pkgs/top-level/all-packages.nix:22922]: This list is sorted. Keep it sorted

**Review by lovesegfault — CHANGES_REQUESTED** [review]
Other than the sorting issue this is AOK

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>mopidy-scrobbler</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107993` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mopidy-scrobbler</li>
  </ul>
</details>

File: maintainers/maintainer-list.nix [hunk 2]
```diff
@@ -10258,4 +10258,10 @@
     github = "zupo";
     githubId = 311580;
   };
+  jakeisnt = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:10261]: Please also sort this.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: maintainers/maintainer-list.nix (hunk 2 above)
Comment (jakeisnt) [reply maintainers/maintainer-list.nix]: Done : ) sorry about that!

File: pkgs/applications/audio/mopidy/default.nix [hunk 3]
```diff
@@ -40,6 +40,7 @@ let
 
     mopidy-tunein = callPackage ./tunein.nix { };
```
Comment (otavio) [inline pkgs/applications/audio/mopidy/default.nix:41]: The `callPackage` addition should also be sorted.

**Review by otavio — CHANGES_REQUESTED** [review]

**Review by otavio — CHANGES_REQUESTED** [review]
Would you mind to sort the packages in a commit before adding the new package? This makes this easier to review and keep the changes on individual and logical commits.

**Comment by jakeisnt** [comment]
> Would you mind to sort the packages in a commit before adding the new package? This makes this easier to review and keep the changes on individual and logical commits.

Sure.

What would you like the commit message to be? 

Would you prefer the commit to sort the packages to be before or after the commit to add mopidy-scrobbler?

**Comment by otavio** [comment]
@jakeisnt I'd probably do it before.

File: pkgs/applications/audio/mopidy/default.nix [hunk 4]
```diff
@@ -14,9 +14,9 @@ let
 
     mopidy-gmusic = callPackage ./gmusic.nix { };
 
-    mopidy-local = callPackage ./local.nix { };
```
Comment (SuperSandro2000) [inline pkgs/applications/audio/mopidy/default.nix:17]: Please put those formatting changes into its own commit.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 107993` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>mopidy-scrobbler</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>mopidy-scrobbler</li>
</ul>
</details>

**Comment by SuperSandro2000** [comment]
Please change ``Reorder mopidy package declarations`` to ``mopidy: sort package declarations``

**Comment by jakeisnt** [comment]
> Please change `Reorder mopidy package declarations` to `mopidy: sort package declarations`

This has been done.

**Review by SuperSandro2000 — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review. If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 107993` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>mopidy-scrobbler</li>
</ul>
</details>
